### PR TITLE
chore: test on Python 3.13

### PR DESCRIPTION
**Pull request by henryiii**
3.13 final is releasing on Tuesday.

### CODE REVIEW

**Comment by henryiii** [comment]
@eduardo-rodrigues, it would be good to get patch releases before 3.13 is released, on Tuesday!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ jobs:
     - uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
+        allow-prereleases: true
```
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply .github/workflows/ci.yml:54]: OK, proceeding towards a merge.

**Review by eduardo-rodrigues — APPROVED** [review]
Sounds good. I will get this merge and will prepare a release.

**Comment by eduardo-rodrigues** [comment]
> @eduardo-rodrigues, it would be good to get patch releases before 3.13 is released, on Tuesday!

Done via https://github.com/scikit-hep/particle/releases/tag/v0.25.2.

**Pull request merged**